Doc/library/sqlite3.rst

```diff
@@ -877,8 +879,9 @@ Cursor Objects
 
    .. method:: execute(sql, parameters=(), /)
 
-      Execute an SQL statement. Values may be bound to the statement using
-      :ref:`placeholders <sqlite3-placeholders>`.
+      Execute SQL statement *sql*.
+      Bind values to the statement using :ref:`placeholders
+      <sqlite3-placeholders>` that map to the parameters in *parameters*.
```

Comment:
This is better, I'm still not entirely certain what form ``parameters`` should take. Just guessing, but is it a tuple (or collection) of strings? Or something else?

Reply:
Well, it can be a sequence or a dict. The params ref all explain this. I'll rewrite it to be more explicit.

Reply:
Addressed in 856a0beb3cd145b4735caccc8f9ac2b8f811a0f5.